metricbeat/docs/modules/http.asciidoc

```diff
@@ -0,0 +1,47 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[metricbeat-module-http]]
+== http Module
+
+Http module is a [Metricbeat](https://www.elastic.co/products/beats/metricbeat) used to call HTTP endpoints.
+Multiple endpoints can be configured which are polled in a regular interval and the result is shipped to the configured output channel.
+
+Httpbeat is inspired by the Logstash [http_poller](https://www.elastic.co/guide/en/logstash/current/plugins-inputs-http_poller.html) input filter but doesn't require that the endpoint is reachable by Logstash as the Metricbeat module pushes the data to the configured outpust channels, e.g. Logstash or Elasticsearch.
+This is often necessary in security restricted network setups, where Logstash is not able to reach all servers. Instead the server to be monitored itself has Metricbeat installed and can send the data or a collector server has Metricbeat installed which is deployed in the secured network environment and can reach all servers to be monitored.
+
+Example use cases are:
```

Comment:
I suggest to remove these examples as we cover these by its own modules (except spring-boot), but that should come as a module soon.